scripts/core/kernel.yml

```diff
@@ -372,6 +382,9 @@ params:
     - type: "uint32_t"
       name: argIndex
       desc: "[in] argument index in range [0, num args - 1]"
+    - type: "const $x_mem_obj_properties_t*"
+      name: pProperties
+      desc: "[in][optional] pointer to Memory object properties."
```

Comment:
I suspect this will break several tests that aren't being built [here](https://github.com/oneapi-src/unified-runtime/tree/main/test/conformance/kernel). I'm not 100% sure on how to do this, but I think you need to set the `UR_DPCXX` cmake variable to point to a build of DPC++. @aarongreig can you advise?

Reply:
done